internal/winapi/cimfs.go

```diff
@@ -32,11 +32,16 @@ type CimFsFileMetadata struct {
 	EACount            uint32
 }
 
+type CimFsImagePath struct {
+	ImageDir  *uint16
+	ImageName *uint16
+}
+
 //sys CimMountImage(imagePath string, fsName string, flags uint32, volumeID *g) (hr error) = cimfs.CimMountImage?
 //sys CimDismountImage(volumeID *g) (hr error) = cimfs.CimDismountImage?
 
 //sys CimCreateImage(imagePath string, oldFSName *uint16, newFSName *uint16, cimFSHandle *FsHandle) (hr error) = cimfs.CimCreateImage?
-//sys CimCloseImage(cimFSHandle FsHandle) = cimfs.CimCloseImage?
+//sys CimCloseImage(cimFSHandle FsHandle) = cimfs.CimCloseImage
```

Comment:
May be good to add `?` to the end here, in case it's attempted on a build without the DLL, it will error instead of panicking.

Reply:
Yeah, I think there is a bug in the `mkwinsyscall` code where if the function has void return and you put a `?`, it will still generate a return value for the syscall. I couldn't find a fix for that after a cursory look, so I changed it here. We can make a fix to `mkwinsyscall` later on and then update this.